docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+services:
+  app:
+    restart: always
+    image: docker-rwd:tag
```

Comment:
It's possible you have an old untagged image hanging around on your system. You can check by doing `docker images | grep rwd`. They can be cleaned up with `docker rmi`. I would confirm that `update.sh` creates a new image labelled as `docker-rwd` (without the tag).